specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchservice.json

```diff
@@ -7657,6 +7657,38 @@
       },
       "description": "A skill to translate text from one language to another."
     },
+    "DocumentExtractionSkill": {
+      "x-ms-discriminator-value": "#Microsoft.Skills.Util.DocumentExtractionSkill",
+      "allOf": [
+        {
+          "$ref": "#/definitions/SearchIndexerSkill"
+        }
+      ],
+      "properties": {
+        "parsingMode": {
+          "type": "string",
+          "x-nullable": true,
```

Comment:
Does the service actually send a JSON `null` in some cases?  Same question applies to all `x-nullable: true` in this change.

Reply:
Ad the documentation mentions https://docs.microsoft.com/en-us/azure/search/cognitive-search-skill-document-extraction if this value is not defined, it will be set to "default" automatically. So I assume this field is nullable, but don't think the service will explicitly send out a  "parsingMode": null

Reply:
`x-nullable:true` means that the service sends/accepts a JSON `null` on the wire, it doesn't necessarily translate to "allow setting this field to null in code" although it might in some languages.  More info [here](https://github.com/Azure/autorest/tree/master/docs/extensions#x-nullable).  Given this, does it change your authoring?

Reply:
The service usually returns explicit nulls (long story, @heaths knows the details).

Reply:
I still think we should keep "x-nullable": true
Also it aligns with other skills we have